fbc/test-fbc/catalog-template.yaml

```diff
@@ -7,9 +7,9 @@ entries:
     name: sandboxed-containers-operator
     schema: olm.package
   - entries:
-      - name: sandboxed-containers-operator.v1.11.0
-        replaces: sandboxed-containers-operator.v1.10.3
-        skipRange: '>=1.1.0 <1.11.0'
+      - name: sandboxed-containers-operator.v1.10.3
```

Comment:
Test catalogs are for the current version in `devel` , which is 1.11.0. We should just ignore them in this PR.